src/main/app.js

```diff
@@ -438,6 +442,18 @@ fluid.defaults("gpii.app", {
     defaultTheme: "white"
 });
 
+// Indicative fix for GPII-3818
+gpii.app.updateKeyedInUserToken = function (that, userToken) {
```

Comment:
Looks like this pull has somehow had #117 incorporated - this is probably fine, but should be highlighted, and we should review how this happened

Reply:
Right, I guess this was incorporated because @stegru needed to properly test the auto key-in feature, for which #117 was needed.